File: Programmers/src/Javascript/Lv1/최대공약수와최소공배수.js
function solution(n, m) {
    let gcd = (a,b) => a%b===0 ? b : gcd(b, a%b);
    let lcm = (a,b) => a*b / gcd(a,b);
    return [gcd(n,m), lcm(n,m)];
}

File: Programmers/src/Java/Lv1/최대공약수와 최소공배수.java
class Solution {
    public int[] solution(int n, int m) {
        int[] answer = new int[2];
        int max = Math.max(n, m);
        int min = Math.min(n, m);
        
        answer[0] = gcd(max, min);
        answer[1] = max*min/answer[0];
        
        return answer;
    }
    
    int gcd(int a, int b) {
        if(a%b==0)
            return b;
        
        return gcd(b, a%b);
    }
}